parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -56,17 +58,19 @@
  */
 public class ThriftSchemaConvertVisitor implements ThriftType.TypeVisitor {
 
-  public FieldProjectionFilter getFieldProjectionFilter() {
-    return fieldProjectionFilter;
-  }
+    private static final Log logger = Log.getLog(ThriftSchemaConvertVisitor.class);
```

Comment:
In fact, this is a problem throughout, which makes the PR hard to review. I'd appreciate it if you could fix this and force-push to update the PR.